### ISSUE TITLE
signify creates infinite highlight groups

### ISSUE THREAD

**Issue by aguthstuditemps** [mhinz/vim-signify#393]
I'm having the same problem as this guy: https://www.reddit.com/r/neovim/comments/qn8s0u/e849_too_many_highlight_syntax_groups/
When using signify, my `:hi` get's filled with
```
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
               xxx cleared
```
and at some point every time I switch buffers I'm getting `E849: Too many highlight and syntax groups`
I'm using neovim `NVIM v0.6.0-dev+566-g4174244d8` right now. Removing this plugin makes the problem go away.

**Comment by jamessan** [comment]
You're running a very old neovim nightly.  Update.  This bug was fixed in neovim almost 2 months ago.

**Comment by aguthstuditemps** [comment]
Oh, I just updated my plugins and didn't think it would be a neovim bug, because I only observed this with signify.
I'm on `NVIM v0.7.0-dev+923-gecec957125` now and the problem is gone. Thanks :) 👍

**Comment by Sangdol** [comment]
For people who want a bit more information like me, [this is the ticket](https://github.com/neovim/neovim/issues/16494) and the fix is not included in [0.6.1](https://github.com/neovim/neovim/releases/tag/v0.6.1).

You need to install the development version if you want to fix this e.g., for Mac ([installing neovim](https://github.com/neovim/neovim/wiki/Installing-Neovim#homebrew-on-macos-or-linux))
```
brew install --HEAD neovim
```
